pandas/core/indexes/base.py

```diff
@@ -1710,6 +1710,42 @@ def _invalid_indexer(self, form, key):
                             kind=type(key)))
 
     def get_duplicates(self):
+        """
+        Extract duplicated index elements.
+
+        This function returns a sorted list of index elements which appear more
+        than once in the index.
+
+        Returns
+        -------
+        array-like
+            List of duplicated indices.
+
+        See Also
+        --------
+        :meth:`Index.duplicated` : Return boolean array denoting duplicates.
+        :meth:`Index.drop_duplicates` : Return Index with duplicates removed.
```

Comment:
In the See Also section, it is not needed to make an explicit link with the `` :meth:`..` `` syntax, you can just use Index.duplicated